### DISCUSSION TITLE
Help: First time contributer

### DISCUSSION THREAD

**Discussion by dvrsaran** [organicmaps/organicmaps#3905] [Q&A]
Hello, I am new to this community and like this project idea, I want to contribute to this project. but I don't know where I start to contribute. I am proficient in Java programming language, YAML, and Spring Boot. please help me to contribute.

**Comment by biodranik** [comment]
Welcome! You can start with README in the root, and then with many docs in the `docs` folder.

**Comment by rtsisyk** [comment]
> Hello, I am new to this community and like this project idea, I want to contribute to this project. but I don't know where I start to contribute. I am proficient in Java programming language, YAML, and Spring Boot. please help me to contribute.

Namaskaram, @dvrsaran. https://github.com/organicmaps/organicmaps/blob/master/docs/CONTRIBUTING.md is a good starting point.
